probe_scraper/parsers/scalars.py

```diff
@@ -7,11 +7,18 @@
 
 
 def extract_scalar_data(s):
+
+    # External scalars.yaml files have release/prerelease, not opt-in/opt-out
+    try:
+        optout = s.dataset.endswith('_OPTOUT')
+    except KeyError:
+        optout = s._definition.get('release_channel_collection', 'prerelease') == 'prerelease'
```

Comment:
Per [the design doc](https://docs.google.com/document/d/10FVEYIFkybRotbe30FquAGE7O0K92bAKSinvrsCfh6o/edit?ts=5af99e01#heading=h.agjq0277ox7i) this should be `collect_on_channels`?

Can we just add a property in `parse_scalars.py` that encapsulates this behavior?

Reply:
I agree, it should be part of `parse_scalars.py`. How about we keep as-is, and I'll make a bug for adding an `is_optout()` function to `parse_scalars.py`, and when that's merged we can update this?